### PR TITLE
Move static aggregation instances out of implementation classes

### LINKED ISSUE #5544: Move static aggregation fields out of aggregation implementations
Many aggregation functions declare a public static field, which is an instance of the implementation. For example: https://github.com/prestodb/presto/blob/c7bf63a2138c2e61f0edb307cefbf6b35fba25c2/presto-main/src/main/java/com/facebook/presto/operator/aggregation/ApproximateCountDistinctAggregations.java#L35

This involves calling a function which creates a `new TypeRegistry()`. This is not a good idea, as it will be missing any types created by plugins. Instead we should remove those static fields, as they're not needed by the Presto server, and move them to the test/benchmarking code that uses them.

### PR DESCRIPTION
**Pull request by geraint0923**
Fix #5544

### CODE REVIEW

File: presto-main/src/test/java/com/facebook/presto/operator/aggregation/AbstractTestApproximateCountDistinct.java [hunk 1]
```diff
@@ -46,6 +47,8 @@
 
     public abstract Object randomValue();
 
+    protected static final MetadataManager metadata = MetadataManager.createTestMetadataManager();
```
Comment (cberner) [inline presto-main/src/test/java/com/facebook/presto/operator/aggregation/AbstractTestApproximateCountDistinct.java]: I would move this to the subclasses that need it, and make it `private`

**Comment by cberner** [comment]
Check why Travis failed. Otherwise looks good

**Comment by electrum** [comment]
This commit message title too long. Please limit titles to ~50 characters.

**Comment by geraint0923** [comment]
@electrum What about `Move static aggregation instances out of implementation classes`?

**Comment by electrum** [comment]
@geraint0923 that's good

**Comment by geraint0923** [comment]
Addressed the comment. Will merge after the tests pass.

**Pull request merged**